src/Identity/UI/src/IdentityBuilderUIExtensions.cs

```diff
@@ -47,7 +47,7 @@ public static IdentityBuilder AddDefaultUI(this IdentityBuilder builder)
         private static readonly IDictionary<UIFramework, string> _assemblyMap =
             new Dictionary<UIFramework, string>()
             {
-                [UIFramework.Bootstrap4] = "Microsoft.AspNetCore.Identity.UI.Views.V4",
+                [UIFramework.Bootstrap4] = "Microsoft.AspNetCore.Identity.UI.",
```

Comment:
I think this goes away. This used to switch the related part but we're not generating any now.

Reply:
Yep. You're right. I should've looked way closer at this. I made the appropriate changes in https://github.com/dotnet/aspnetcore/pull/31039.